src/webserver.cpp

```diff
@@ -673,9 +673,10 @@ int webserver::bodyfull_requests_answer_first_step(
         )
     )
     {
+        const size_t post_memory_limit (32*1024);  // Same as #MHD_POOL_SIZE_DEFAULT
         mr->pp = MHD_create_post_processor (
                 connection,
-                1024,
+                post_memory_limit,
```

Comment:
Unfortunately, MHD_POOL_SIZE_DEFAULT is only defined in src/microhttpd/daemon.c.  So it is not visible to library users.